flexmeasures/utils/config_utils.py

```diff
@@ -71,12 +71,27 @@ def check_app_env(env: str | None):
 
 def read_config(app: Flask, custom_path_to_config: str | None):
     """Read configuration from various expected sources, complain if not setup correctly."""
+    flexmeasures_env = "production"
+    if app.testing:
+        flexmeasures_env = "testing"
+    elif os.getenv("FLEXMEASURES_ENV", None):
+        flexmeasures_env = os.getenv("FLEXMEASURES_ENV", None)
+    else:
+        flexmeasures_env = os.getenv("FLASK_ENV", None)
+        import warnings
+
+        warnings.warn(
+            "'FLASK_ENV' is deprecated and will be removed in FlexMeasures X.X.X"
+            " Change FLASK_ENV to FLEXMEASURES_ENV in the environment variables",
```

Comment:
Just putting it out there so we considered it: what about having our server set the new environment variable here:
```
import os
os.environ["FLEXMEASURES_ENV"] = os.environ.pop("FLASK_ENV")
```
with a corresponding warning? Or maybe keep the original environment variable just to be safe:
```
import os
os.environ["FLEXMEASURES_ENV"] = flexmeasures_env
```

Reply:
It's probably better to leave it as is and fallback to FLASK_ENV if FLEXMEASURES_ENV is not set. It's a bit weird if the FLASK_ENV namespace is used by FlexMeasures but there is also no real problem with it.